WowUp.Common/Models/Curse/CurseDependencyType.cs

```diff
@@ -0,0 +1,28 @@
+﻿using WowUp.Common.Enums;
+
+namespace WowUp.Common.Models.Curse
+{
+    public enum CurseDependencyType
+    {
+        EmbeddedLib = 1,
+        Optional = 2,
+        Required = 3,
+        Tool = 4,
+        Incomplete = 5,
```

Comment:
Not relevant to WoW but this should be 'Incompatible', not 'Incomplete'.

Reply:
ty, resolved it now.